plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -188,6 +193,44 @@ def __cmp__(self, other):
             other.complete_version_serialized
         )
 
+    def get_or_calculate_checksum(self, checksumtype):
```

Comment:
Maybe capture the side effect in the name by using `get_or_calculate_and_save_checksum`? I'm not totally sold on this idea, but I want to mention it.